packages/i18nify-js/src/modules/currency/__tests__/mocks/formatNumberToParts.ts

```diff
@@ -0,0 +1,1167 @@
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+      { type: 'decimal', value: '.' },
+      { type: 'fraction', value: '33' },
+    ],
+  },
+  XAF: {
+    currency: 'FCFA',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'FCFA' },
+      { type: 'literal', value: `${nbsp}` },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+    ],
+  },
+  CNY: {
+    currency: 'CN¥',
+    decimal: '.',
+    fraction: '33',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'CN¥' },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+      { type: 'decimal', value: '.' },
+      { type: 'fraction', value: '33' },
+    ],
+  },
+  EGP: {
+    currency: 'EGP',
+    decimal: '.',
+    fraction: '33',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'EGP' },
+      { type: 'literal', value: `${nbsp}` },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+      { type: 'decimal', value: '.' },
+      { type: 'fraction', value: '33' },
+    ],
+  },
+  FKP: {
+    currency: 'FK£',
+    decimal: '.',
+    fraction: '33',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'FK£' },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+      { type: 'decimal', value: '.' },
+      { type: 'fraction', value: '33' },
+    ],
+  },
+  LBP: {
+    currency: 'LBP',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'LBP' },
+      { type: 'literal', value: `${nbsp}` },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+    ],
+  },
+  SSP: {
+    currency: 'SSP',
+    decimal: '.',
+    fraction: '33',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'SSP' },
+      { type: 'literal', value: `${nbsp}` },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+      { type: 'decimal', value: '.' },
+      { type: 'fraction', value: '33' },
+    ],
+  },
+  WST: {
+    currency: 'WS$',
+    decimal: '.',
+    fraction: '33',
+    integer: '123,456',
+    isPrefixSymbol: true,
+    rawParts: [
+      { type: 'currency', value: 'WS$' },
+      { type: 'integer', value: '123' },
+      { type: 'group', value: ',' },
+      { type: 'integer', value: '456' },
+      { type: 'decimal', value: '.' },
+      { type: 'fraction', value: '33' },
+    ],
+  },
+};
+
+export const negativeNumberPartsIntlMap = {
```

Comment:
Why do we need this to store statically ? 
Can this not be generated at runtime ? basically add whatever is required in normal postive config ?